### PR TITLE
added kr2484.txt

### PR DESCRIPTION
**Pull request by karimerobles**

### CODE REVIEW

**Comment by timkpaine** [comment]
which one would you like to start with?

**Comment by karimerobles** [comment]
Neither! I’ve settled on collaborating with another student on a iOS Quiz-type game app.

File: kr2484.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Project ideas:
+
+1. Saw this idea online, don't think anyone else has done it: A plug-in or something that reformats recipes from their standard format (ingredients/instructions) into a more digestible/intuitive format that makes it easier as you're following along (qtys incorporated into steps)
```
Comment (timkpaine) [inline kr2484.txt]: need to update proposal

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: kr2484.txt (hunk 1 above)
Comment (karimerobles) [reply kr2484.txt]: Proposal has been updated.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# project-proposals-f2020
-Project Proposals for Fall 2020
+# QuickQuestions
```
Comment (timkpaine) [inline README.md:1]: don't overwrite my readme

File: kr2484.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Proposal:
```
Comment (timkpaine) [inline kr2484.txt]: follow file naming guidelines

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Pull request closed without merging**